composer.json

```diff
@@ -1,10 +1,55 @@
 {
-    "name"       : "automattic/msm-sitemap",
-    "description": "Comprehensive sitemaps for your WordPress.com VIP site.",
-    "homepage"   : "https://vip.wordpress.com/plugins/msm-sitemap/",
-    "type"       : "wordpress-plugin",
-    "license"    : "GPL-2.0+",
-    "require"    : {
-        "composer/installers": "~1.0"
+  "name": "automattic/msm-sitemap",
```

Comment:
With the `.editorconfig` fix to put JSON back to having a 4-space tab indent, this file will mostly be fixed.

That said, I highly recommend installing https://github.com/ergebnis/composer-normalize locally, and running `composer normalize` so that it formats, and re-orders the keys in a consistent way.